ui/_webSrc/.gitignore

```diff
@@ -0,0 +1,14 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
+
+node_modules
+dist
+dist-ssr
+*.local
+
```

Comment:
left some of the files that I thought were more front end app specific in the inner ignore file. 

some of these could be removed or moved.